### PR TITLE
Fixes for reported issues #4 and #5

### PR DESCRIPTION
**Pull request by batkinson**
Hi,

This request includes fixes for issues that I reported previously (listed in the subject). We're actively using the patched plugin at trac.openxdata.org if you care to see how it's working.  BTW, thank you for the great plugin, we're loving it.

### CODE REVIEW

File: backlog/prefs.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2009 John Szakmeister
```
Comment (jszakmeister) [inline backlog/prefs.py]: Give credit where credit is due Brent.  That should be your name, with the current year.

File: backlog/web_ui.py [hunk 2]
```diff
@@ -197,8 +208,9 @@ def __init__(self):
         data['form_token'] = req.form_token
         data['active_milestones'] = self._get_active_milestones(milestone)
         data['base_path'] = req.base_path
+        data['shown_fields'] = req.session.get('backlog_fields') or self._ticket_fields
 
-        if 'TICKET_MODIFY' in req.perm:
+        if mod_perms <= req.perm:
```
Comment (jszakmeister) [inline backlog/web_ui.py]: Thinking about this more, we should probably only care about 'BACKLOG_ADMIN' here.  If the goal is to split out permissions, then I don't see why we need TICKET_MODIFY in here at all.  If we were going to allow either TICKET_MODIFY or BACKLOG_ADMIN, that'd be one thing.  But this doesn't do that.  So, I think here should be:

``` python
if 'BACKLOG_ADMIN' in req.perm:
```

And I think the block above should just be:

``` python
req.perm.require('BACKLOG_ADMIN')
```

And you can drop mod_perms all together.  Does that sound reasonable to you?

**Comment by jszakmeister** [comment]
Thanks Brent.  I think if you can make a couple of minor adjustments, I can merge this in.  Thanks for taking the time to put this together.  I've been completely swamped with kernel and compiler bugs on another project I'm working on.

**Comment by batkinson** [comment]
Hi, I pushed two commits that incorporated your feedback. Thanks for taking a look and for the feedback.

**Comment by jszakmeister** [comment]
Thank you!

**Pull request merged**